us/ca/cities/san-mateo/drafting-manual/index.xml

```diff
@@ -0,0 +1,484 @@
+      </para>
+    </para>
+    <para>
+      <num>(b)</num>
+      <heading>Chapters.</heading>
+      <para>
+        <num>(1)</num>
+        <text>Chapters shall be numbered with two zero-padded sequential Arabic numerals: "Chapter &lt;title num&gt;.&lt;chapter num&gt;".</text>
+      </para>
+      <para>
+        <num>(2)</num>
+        <text>A title case title is required without a trailing period.</text>
+      </para>
+      <para>
+        <num>(3)</num>
+        <text>Chapter num and title shall be centered and bold.</text>
+      </para>
+      <para>
+        <num>(4)</num>
+        <text>Chapters shall not include a table of contents.</text>
+      </para>
+    </para>
+    <para>
+      <num>(c)</num>
+      <heading>Articles.</heading>
+      <para>
+        <num>(1)</num>
+        <text>Articles shall be numbered with two zero-padded sequential Arabic numerals: "Article &lt;article num&gt;".</text>
+      </para>
+      <para>
+        <num>(2)</num>
+        <text>A title case title is required without a trailing period.</text>
+      </para>
+      <para>
+        <num>(3)</num>
+        <text>Article num and title shall be centered and bold.</text>
+      </para>
+    </para>
+    <para>
+      <num>(d)</num>
+      <heading>Sections.</heading>
+      <para>
+        <num>(1)</num>
+        <text>Sections shall be numbered with three zero-padded Arabic numerals in sequential multiples of ten: "&lt;title num&gt;.&lt;chapter num&gt;.&lt;section num&gt;".</text>
+      </para>
+      <para>
+        <num>(2)</num>
+        <text>A title case title is required without a trailing period.</text>
+      </para>
+      <para>
+        <num>(3)</num>
+        <text>Section num and title shall be bold.</text>
+      </para>
+      <para>
+        <num>(4)</num>
+        <text>Section text shall start on a new line following the title.</text>
+      </para>
+    </para>
+    <para>
+      <num>(e)</num>
+      <heading>Citations.</heading>
+      <para>
+        <num>(1)</num>
+        <text>There shall be no reference in the Code to an Ordinance.</text>
+      </para>
+      <para>
+        <num>(2)</num>
+        <text>Statutes of the State or Federal government may be referred to when, and only when, they are not to be codified.  For example, the City's tidelands trust grant is in a statute that will not be codified; however, laws regarding State redevelopment will almost always be codified in either the <cite doc="Cal. Gov't Code" proof="needed">Government Code</cite> or the <cite doc="Cal. Health &amp; Safety Code" proof="needed">Health and Safety Code</cite>.</text>
+      </para>
+    </para>
+  </section>
+  <section>
+    <num>4</num>
+    <heading>Ordinance guidelines.</heading>
+    <text/>
+    <para>
+      <num>(a)</num>
+      <text>Banner and Title.</text>
+      <para>
+        <num>(1)</num>
+        <text>The banner and title shall be in the following form:</text>
+        <include>
+          <text>
```

Comment:
hmm we probably want to update this to `<text class="center">CITY OF SAN MATEO</text>`, and other similar occurences